src/components/menu/menu.tsx

```diff
@@ -83,6 +90,12 @@ export class Menu {
         this.portalId = createRandomString();
     }
 
+    public componentDidLoad() {
+        if (!this.host.querySelector('[slot="trigger"]')) {
+            console.warn('Using limel-menu with the default trigger is deprecated. Please provide your own trigger element.');
```

Comment:
Improved examples, not using the default trigger element, are coming in another branch.